develop/app/Config/Filters.php

```diff
@@ -19,6 +20,7 @@ class Filters extends BaseConfig
 		'csrf'     => CSRF::class,
 		'toolbar'  => DebugToolbar::class,
 		'honeypot' => Honeypot::class,
+		'user' => UserFilter::class,
```

Comment:
You should pay more attention to indentation, and command the `php-cs-fixer`.

Reply:
This chahge is done.